### DISCUSSION TITLE
Winpilot Bloatless 2024.5.6

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#201] [Announcements]
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Adblock for the Windows 11 UI now integrated into Winpilot. Sounds strange, but it&#39;s also unfortunately a strange development on <a href="https://twitter.com/Microsoft?ref_src=twsrc%5Etfw">@Microsoft</a>&#39;s part.<a href="https://t.co/LrKCWINsLl">https://t.co/LrKCWINsLl</a><a href="https://twitter.com/hashtag/winpilot?src=hash&amp;ref_src=twsrc%5Etfw">#winpilot</a> <a href="https://twitter.com/hashtag/Windows11?src=hash&amp;ref_src=twsrc%5Etfw">#Windows11</a> <a href="https://twitter.com/hashtag/Ads?src=hash&amp;ref_src=twsrc%5Etfw">#Ads</a> <a href="https://twitter.com/hashtag/adblock?src=hash&amp;ref_src=twsrc%5Etfw">#adblock</a> <a href="https://twitter.com/hashtag/clippy?src=hash&amp;ref_src=twsrc%5Etfw">#clippy</a> <a href="https://t.co/0b1cvtSa5F">pic.twitter.com/0b1cvtSa5F</a></p>&mdash; Belim (@builtbybel) <a href="https://twitter.com/builtbybel/status/1786408547384783336?ref_src=twsrc%5Etfw">May 3, 2024</a></blockquote> 

- **Winpilots and Clippy (NeoClippy) have shifted their focus towards even stricter debloating and disabling of undesired Microsoft features aligning with the approach found in the Windows 11 OOBE.  Any features not enabled during the initial setup can be adjusted later in Winpilots' setup area.**
- The Workspace offers four different startup options (see screenshot)
- **Support for the [Tiny11Builder](https://github.com/ntdevlabs/tiny11builder) has also been added as a pilot function. It is a partial GUI for the well-known PowerShell script by [NTDEV](https://twitter.com/NTDEV_), With Winpilots' NeoClippy, you can easily download, mount, and prepare Tiny11.iso images. Feel free to give it a try. Please note, though, this is just a test version. Since it's not an official Microsoft build, I can't offer support for its use. Therefore, use of the script is at your own risk. However, it can serve the purpose of testing and evaluation.**
- Some bugs in the interpretation of links in the Clippy bubble have been fixed. 
- License information has been updated, especially regarding Clippy. Even though Clippy is long gone, we still need to respect the rights associated with it, so you'll see a small copyright notice beneath @Microsoft Clippy's search bar.

![jBrjQrRwCI](https://github.com/builtbybel/Winpilot/assets/57478606/49903786-49c7-44c8-8830-dbb13091953a)

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Winpilot/releases/tag/2024.5.6'>Winpilot Bloatless 2024.5.6</a>.</em>

**Comment by ed-br** [comment]
Amazing Man, thank you!!!
